components/feature/push/README.md

```diff
@@ -0,0 +1,21 @@
+# [Android Components](../../../README.md) > Feature > Push
+
+A compoennt that implements push notifications with a supported push service.
+
+## Usage
+
+TBD
```

Comment:
nit: remove this or add something here, or file a follow-up issue and link it here